README.md

```diff
@@ -1,72 +1,57 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Awesome books with ES6.
 
-> Description the project.
+> Hello! I am a software developer! I can help you build a product, feature or website. Tak a look of my works.
+If you like what you see and have a project you need coded, don’t hesitate and contact me.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML
+- CSS
+- JavaScript
 
 ## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link]()
 
 
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
 
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
```

Comment:
- [ ] Please it will be nice to add a `Getting Started` section in your `Readme file` where you outline explicit steps to clone and run your app locally, as this will be of help to non technical users and some of them are potential recruiters.